tide/core/scene/DisplayGroup.h

```diff
@@ -146,6 +151,10 @@ class DisplayGroup : public Rectangle,
     /** Remove a window from the set of focused windows. */
     void removeFocusedWindow( ContentWindowPtr window );
 
+
+    /** Get the set of focused windows. */
```

Comment:
ouch! No that's a copy-paste error, thanks for noticing. Right now the only panel is the Launcher. I think there could be more in the future, like a "Help" panel that could be displayed when the wall is empty or on a tapAndHold on the background, but let's see. Even if there is only one, I find it cleaner to have this concept rather than having methods such as: "isLauncher()", "getLauncherWindow()", etc..